gen/dcos-config.yaml

```diff
@@ -3036,7 +3036,11 @@ package:
           # Flush
           # =====
           # Set the flush interval in seconds.
-          Flush        5
+          Flush        1
+          # Storage
+          # =======
+          # Configure buffer path
+          storage.path              /run/flb-storage/
```

Comment:
shouldn't this be something like {{/var/run/dcos/...}} ?

Reply:
I followed exhibitor config https://github.com/dcos/dcos/blob/e0d8908efbfc757c8c3cd069d33f6f89afbe9a93/packages/exhibitor/extra/dcos-exhibitor.service#L13

Reply:
https://github.com/dcos/dcos/blob/e7db6dfb77156d78e06257b211b4cf78d30dac1c/packages/exhibitor/extra/start_exhibitor.py#L92

Reply:
We should use `/run/dcos` or `/var/lib/dcos` as the base of any new paths, to ensure that all DC/OS data is in a few known places.

Exhibitor is a bit special because it can't run rely on `bootstrap.py` to create these top-level directories since `bootstrap.py` depends on ZooKeeper. And I'd consider working around that.

Reply:
OK, I'll fix it

Reply:
OK, fixed